docs/Dockerfile

```diff
@@ -4,21 +4,14 @@ WORKDIR /workspace
 COPY content content
 COPY tools tools
 COPY Makefile ./
+RUN wget -O busybox https://busybox.net/downloads/binaries/1.31.0-defconfig-multiarch-musl/busybox-`arch` && chmod +x busybox
```

Comment:
Maybe I'm getting older but I would resist this kind of thing - downloading an executable from the internet outside of well-known package repositories and not performing any kind of signature fingerprint validation (is `busybox` signed at all?)

An alternative approach is to use nginx to host docs. `nginx` is available as a package on `ubi8-minimal`.

Not quite the same, but we use nginx container for docs downstream https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-docs/Dockerfile.in

Apart from that, the initiative to remove product docs built in the product and as a stand-alone container is scheduled for 3.74.0 - https://issues.redhat.com/browse/ROX-12465
Maybe we don't need this change at all?

Reply:
I need this to release 3.73 due to known vulns. I agree that downloading something from internet is not the best option unfortunately I can't find busybox on ubi minimal :(
Ideal solution will be to use official busybox image.

Reply:
Since it's going to be removed maybe removing problematic packages solves the problem.